packages/yourloops/test/unit/pages/notifications/notification.test.tsx

```diff
@@ -29,7 +29,6 @@ import React from 'react'
 import moment from 'moment-timezone'
```

Comment:
Tests line 207 and 213 could be removed from this file, there are now tested in integration